### DISCUSSION TITLE
linux: close tab button losing focus

### DISCUSSION THREAD

**Discussion by Technetium1** [ghostty-org/ghostty#3285] [Feature Requests, Ideas] [answered]
If you try to close multiple tabs quickly with the close tab button, focus on the button is lost. This leads to subsequent clicks being treated as titlebar clicks, and surprising you by minimizing or maximizing.

Just installed with no custom config [ghostty 1.0.0-2](https://archlinux.org/packages/extra/x86_64/ghostty/)

**Comment by mitchellh** [comment]
What distro?

Can you record a video showing what's going on?

**Reply by Technetium1** [reply]
Garuda with KDE
[closebutton.webm](https://github.com/user-attachments/assets/da0732f3-221b-4dad-b86d-0e2e978a3c34)

**Comment by jcollie** [comment, marked as answer]
This is likely a GTK issue, as we use GTK to manage tab headers and the rest of the native GUI elements.

**Reply by Technetium1** [reply]
Using [4.16.7-3](https://archlinux.org/packages/extra/x86_64/gtk4/)
